src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1414,7 +1414,7 @@ void Graph::DropNode(const NodePtr &node) {
             if (!child)
```

Comment:
May I ask you to fix code on the lines L1411-L1413 as well?
weak_ptr must be locked and validated just once, like on L1405, otherwise it can be invalidated in other thread.
The same is applicable for fn `DropDWConvNode`.

Reply:
Corrected, please double check. Thanks